src/systems/ec/implementations/vanilla/packages/ec-component-table/ec-component-table.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable unicorn/consistent-function-scoping */
+import { queryOne, queryAll } from '@ecl/ec-base/helpers/dom';
+
+/**
+ * @param {HTMLElement} element DOM element for component instantiation and scope
+ * @param {Object} options
+ * @param {String} options.sortSelector Selector for toggling element
+ * @param {Boolean} options.attachClickListener Whether or not to bind click events on toggle
```

Comment:
I don't think this one should be optional. If it is off, then the behavior will be unnecessarily loaded without being able to do anything.

Reply:
You're right the accordion(s) and the expandable have this option to optionally attach a listener or not. My comment is related this https://github.com/ec-europa/europa-component-library/pull/1598/files#diff-9c8d63a91d8421e5c83211d9d55da702R93 
Is it really that the user adding javascript to sort values in the table will want to put `false` on this flag to not be able to use the behavior? I don't see possible use cases for such options.

Reply:
Ok sure it make sense now, thx for explanation, code updated ;)